### PR TITLE
WCSim exectuable will exit if jobOptions.mac, jobOptions2.mac, or tun…

### PR DESCRIPTION
**Pull request by tdealtry**
…ing_parameters.mac are not found. For issue #85

### CODE REVIEW

**Comment by cwwalter** [comment]
As I mentioned I don't think we want to require vis.mac.

File: WCSim.cc [hunk 1]
```diff
@@ -114,6 +126,7 @@ int main(int argc,char** argv)
     G4String command = "/control/execute ";
     G4String fileName = argv[1];
 
+    file_exists(fileName.c_str());
```
Comment (cwwalter) [inline WCSim.cc]: What happens if you just run WCSim with a non-existent file name on the command line now?  I thought it already didn't work...

**Comment by tdealtry** [comment]
I made the commit, but perhaps github doesn't make this a notification? I think this is now ready to merge.

**Comment by eosullivan** [comment]
You usually don't get a notification for your own commits, although in the case of this commit I didn't get a notification either. Anyways, I checked out this pull request branch and it works, so I will merge this in. Thanks!

**Pull request merged**